docs/core/testing/unit-testing-mstest-runner-integrations.md

```diff
@@ -0,0 +1,58 @@
+---
+title: MSTest runner integrations
+description: Learn how to run MSTest runner tests through MSBuild and dotnet test.
+author: nohwnd
+ms.author: jajares
+ms.date: 12/20/2023
+---
+
+# MSTest runner integrations
+
+MSTest runner integrates with MSBuild and `dotnet test` to provide a simple way to run all tests in a solution (`.sln`) or in a single project.
+
+## `dotnet test` integration
+
+### `dotnet test` integration - VSTest mode
+
+MSTest runner provides a compatibility layer to work with `dotnet test` seamlessly. This layer requires `Microsoft.Testing.Platform.MSBuild` NuGet package. This package is automatically installed as a dependency of `MSTest` and `MSTest.TestAdapter` package.
+
+Tests can be executed by running:
+
+```
+dotnet test
+```
+
+This layer runs test through VSTest and integrates with it on VSTest Test Framework Adapter level.
+
+### `dotnet test` - MSTest runner mode
+
+By default VSTest is used to execute MSTest runner tests. User can opt-in to a fully MSTest runner provided mode. This mode integrates directly with the `dotnet test` target at MSBuild target level. The user opts-in by specifying `<TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>` in their project file. When this mode is enabled, VSTest is not used to run tests at all.
```

Comment:
You shouldn't say "User" here, because you're talking to the reader directly. They're here to learn something so you can address them directly. Also, unless needed to clarify something, you should use the word `run` instead of `execute` as `execute` can also mean to kill a person:
```suggestion
By default, VSTest is used to run MSTest runner tests. You can opt-in to a fully MSTest runner provided mode. This mode integrates directly with the `dotnet test` target at MSBuild target level. You opt-in by specifying `<TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>` in your project file. When this mode is enabled, VSTest is not used to run tests at all.
```
You sort of repeat "opt-in" again, so here's a different suggestion merging those two:
```suggestion
By default, VSTest is used to run MSTest runner tests. You can enable a full MSTest runner by specifying the `<TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>` setting in your project. This setting disables VSTest.
```
And you should provide the XML here too.

Reply:
applied the second option and added xml